### PR TITLE
Add a toggle for intellisense to the sql editor

### PR DESCRIPTION
**Pull request by saltcod**
Allow Intellisense to be toggled in the SQL Editor

screenshot-2024-02-28-at-15 04 25

### CODE REVIEW

**Review by jordienr — APPROVED** [review]

File: apps/studio/components/interfaces/SQLEditor/MonacoEditor.tsx [hunk 1]
```diff
@@ -138,6 +143,36 @@ const MonacoEditor = ({
         // - https://github.com/microsoft/monaco-editor/issues/2229
         // - https://github.com/microsoft/monaco-editor/issues/2503
         // fixedOverflowWidgets: true,
+        suggest: {
```
Comment (saltcod) [inline apps/studio/components/interfaces/SQLEditor/MonacoEditor.tsx:146]: this is all a bit crazy.... I don't see a single option to toggle them all though.

Comment (MildTomato) [reply apps/studio/components/interfaces/SQLEditor/MonacoEditor.tsx:146]: ha

File: apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityActions.tsx [hunk 2]
```diff
@@ -29,13 +29,33 @@ const UtilityActions = ({
...
       <SavingIndicator id={id} />
 
+      <Toggle_Shadcn
+        aria-label="Toggle Intellisense"
+        size="sm"
+        pressed={intellisenseEnabled}
+        className="text-xs font-normal h-auto group"
+        onPressedChange={() => setIntellisenseEnabled(!intellisenseEnabled)}
+      >
+        <div className="flex items-center gap-2">
```
Comment (MildTomato) [inline apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityActions.tsx:48]: this looks like a random div

File: apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityActions.tsx [hunk 3]
```diff
@@ -29,13 +29,33 @@ const UtilityActions = ({
...
+          <span
+            className={cn(
+              'w-2 h-2 flex  rounded-full transition-colors ',
+              intellisenseEnabled ? 'bg-green-900' : 'bg-surface-400 group-hover:bg-green-600'
```
Comment (MildTomato) [inline apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityActions.tsx:52]: bg-green-900 is an old class.

bg-brand please

**Review by joshenlim — APPROVED** [review]

**Pull request merged**